decidim-core/spec/system/account_spec.rb

```diff
@@ -86,6 +86,49 @@
       end
     end
 
+    describe "when updating the user's nickname" do
+      it "changes the user's nickname - 'nickname'" do
+        within "form.edit_user" do
+          fill_in "Nickname", with: "nickname"
+          find("*[type=submit]").click
+        end
+
+        expect(page).to have_content("Your account was successfully updated.")
+        expect(page).to have_field("user[nickname]", with: "nickname", type: "text")
+      end
+
+      it "shows error when word is too long - 'nicknamenicknamenickname'" do
```

Comment:
This is not the case according to the expect (line 106), right? Maybe this example isn't necessary as we already have the character counter and the `maxlength` attribute that's preventing this example?

Reply:
Yeah . This example is here for that "maxlength".

Reply:
Oh I missed that. The thing is that the example isn't showing an error (it says literally "You account was successfully updated."). I'm proposing to correct the example description at https://github.com/decidim/decidim/pull/14405/files#r2028426812